en/identity-server/next/docs/deploy/configure/databases/carbon-database/change-to-mysql.md

```diff
@@ -78,6 +78,29 @@ A sample configuration is given below.
         password = "regadmin"
         ```  
 
+
+    !!! note
```

Comment:
Structure the note similar to MSSQL note. (Replicate the same)

Reply:
https://github.com/wso2/docs-is/pull/4209/commits/d0b97b1a597730a4d099c4dc1565e31452c0f03f